datasets/SCCS/convert.py

```diff
@@ -0,0 +1,155 @@
+# coding: utf8
+from __future__ import unicode_literals, print_function, division
+from collections import Counter, defaultdict, OrderedDict
+
+import attr
+from clldutils.path import Path, read_text, write_text
+from clldutils.dsv import reader, UnicodeWriter
+from clldutils.misc import nfilter
+from pyglottolog.references import BibFile
+
+from pydplace.api import Society
+
+#
+# read wnai.bib
+# write mapping of bibkeys to bib.json
+# read sources.bib
+# merge records from wnai.bib not yet in sources.bib
+# write sources.bib
+#
+
+
+def read_win1252(fname):
+    with open(fname, 'rb') as fp:
+        c = fp.read()
+
+    with open(fname, 'wb') as fp:
+        fp.write(c.replace(b'\x9d', b''))
+
+    for r in reader(fname, dicts=True, encoding='cp1252'):
+        if (r.get('dataset') == 'SCCS') or (r.get('Dataset') == 'SCCS') or (r.get('Datset') == 'SCCS'):
+            yield r
+
```

Comment:
Eww cp1252. Could we just convert the file once to utf8 and store that in the repository? Much nicer to have utf8 everywhere?

Reply:
...and that's what the rest of the code is doing, Simon. Nevermind.

Reply:
Yes, what's stored in the repos is the output of this script.